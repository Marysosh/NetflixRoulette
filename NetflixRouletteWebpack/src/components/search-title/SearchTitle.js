import React from "react";
import "./SearchTitle.scss";

function SearchTitle() {
  return <h2 className="search-title">Find your movie</h2>;
}

export default SearchTitle;
