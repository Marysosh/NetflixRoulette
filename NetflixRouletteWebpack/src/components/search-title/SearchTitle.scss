.search-title {
  color: #ffffff;

  font-family: "Montserrat";
  font-style: normal;
  font-weight: 300;
  font-size: 40px;
  line-height: 49px;

  letter-spacing: 1px;
  text-transform: uppercase;
}
