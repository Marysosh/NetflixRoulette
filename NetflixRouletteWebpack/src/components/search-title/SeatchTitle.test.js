import React from "react";
import { render } from "@testing-library/react";

import SearchTitle from "./SearchTitle";

describe("Search title component", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<SearchTitle />);

    expect(asFragment()).toMatchSnapshot();
  });
});
