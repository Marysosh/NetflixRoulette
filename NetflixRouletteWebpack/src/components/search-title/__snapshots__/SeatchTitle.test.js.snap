// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search title component renders correctly 1`] = `
<DocumentFragment>
  <h2
    class="search-title"
  >
    Find your movie
  </h2>
</DocumentFragment>
`;
